spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/notify/MicrosoftTeamsNotifier.java

```diff
@@ -26,6 +33,7 @@
 	private static final String MANAGEMENT_URL_KEY = "Management URL";
 	private static final String SOURCE_KEY = "Source";
 	private RestTemplate restTemplate = new RestTemplate();
+	private Message message = null;
```

Comment:
Must not add state to the class. It will cause problems when doing concurrent notifications.

Reply:
Ok - reverted this